nix/cardano/packages/default.nix

```diff
@@ -20,6 +20,7 @@ let
     ;
   inherit (inputs.cells) cardano;
   inherit (nixpkgs) lib;
+  cardanoLib = import "${iohk-nix}/cardano-lib/default.nix" {inherit (nixpkgs) lib writeText runCommand jq;};
```

Comment:
See question about re-importing iohk-nix here. And why not using iohk-nix's exported cardanoLib?